{
    "type": "bundle",
    "id": "bundle--c2a4f730-4252-4f2e-b87b-177654294d78",
    "spec_version": "2.0",
    "objects": [
        {
            "object_marking_refs": [
                "marking-definition--fa42a846-8d90-4e51-bc29-71d5b4802168"
            ],
            "id": "relationship--f9a82ef5-524a-4b95-a9b5-560e61f3d2fa",
            "type": "relationship",
            "created": "2021-02-25T01:55:40.902Z",
            "created_by_ref": "identity--c78cb6e5-0c4b-4611-8297-d1b8b55e40b5",
            "external_references": [
                {
                    "source_name": "Cybereason Conti Jan 2021",
                    "url": "https://www.cybereason.com/blog/cybereason-vs.-conti-ransomware",
                    "description": "Rochberger, L. (2021, January 12). Cybereason vs. Conti Ransomware. Retrieved February 17, 2021."
                },
                {
                    "source_name": "CarbonBlack Conti July 2020",
                    "url": "https://www.carbonblack.com/blog/tau-threat-discovery-conti-ransomware/",
                    "description": "Baskin, B. (2020, July 8). TAU Threat Discovery: Conti Ransomware. Retrieved February 17, 2021."
                }
            ],
            "modified": "2021-02-25T01:55:40.902Z",
            "description": "[Conti](https://attack.mitre.org/software/S0575) has used API calls during execution.(Citation: Cybereason Conti Jan 2021)(Citation: CarbonBlack Conti July 2020) ",
            "relationship_type": "uses",
            "source_ref": "malware--4dea7d8e-af94-4bfb-afe4-7ff54f59308b",
            "target_ref": "attack-pattern--391d824f-0ef1-47a0-b0ee-c59a75e27670",
            "x_mitre_version": "1.0",
            "x_mitre_modified_by_ref": "identity--c78cb6e5-0c4b-4611-8297-d1b8b55e40b5"
        }
    ]
}